[Bug]: Atomic Angular not building with angular 20

### Which product are you using?

Atomic

### product version

3.10.0

### bug description

When building an application using angular 20 using `@coveo/atomic-angular` v3.10.0, the following error is thrown :

```
[ERROR] Could not resolve "@coveo/atomic/custom-elements-manifest"


node_modules/@coveo/atomic-angular/fesm2022/coveo-atomic-angular.mjs:8:35:  
8 │ import customElementsManifest from '@coveo/atomic/custom-elements-manifest'; 
  ╵                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The module "./custom-elements.json" was not found on the file system:


node_modules/@coveo/atomic/package.json:42:34:  
42 │     "./custom-elements-manifest": "./custom-elements.json" 
    ╵                                   ~~~~~~~~~~~~~~~~~~~~~~~~
You can mark the path "@coveo/atomic/custom-elements-manifest" as external to exclude it from the
bundle, which will remove this error and leave the unresolved path in the bundle.
```

I tried to exclude it from the bundle by marking it as external (added in `externalDependencies` in angular.json) but it does not fix the issue. It compiles but raise an exception in the browser since the module cannot be found.

Other version might be impacted

### Steps to reproduce

Create a project using angular 20 and `@coveo/atomic-angular` 3.10, add a component using atomic-angular, and run `ng build`

### Relevant log output

```shell

```

--- a/packages/atomic/package.json
+++ b/packages/atomic/package.json
@@ -42,6 +42,7 @@
     "./custom-elements-manifest": "./custom-elements.json"
   },
   "files": [
+    "custom-elements.json",
     "dist/",
     "docs/",
     "licenses/"

